src/routes/dictionary/__tests__/DictionaryScreen.spec.tsx

```diff
@@ -10,6 +10,7 @@ import { getReturnOf } from '../../../testing/helper'
 import render from '../../../testing/render'
 import DictionaryScreen from '../DictionaryScreen'
 
+jest.mock('@react-navigation/native')
```

Comment:
Omitting them fails all tests in the respective testsuite because FeedbackBadge relies on `useIsFocused` which requires the component to be wrapped in a `NavigationContainer`. That's always the case in the actual app, but not inside these unit tests.

The same thing has been done in a number of other files already, and it appears to be the preferred solution to the problem.

Reply:
I see, thanks for explanation.